app/renderer/js/pages/preference/new-server-form.ts

```diff
@@ -56,6 +56,7 @@ export function initNewServerForm({$root, onChange}: NewServerFormProps): void {
     "input.setting-input-value",
   )!;
 
+<<<<<<< HEAD
```

Comment:
Can you remove this merge conflict marker?

Reply:
I think perhaps you remove this in the second commit; in general it's worth reviewing each individual commit with a took like `gitk --all` for readability.

Reply:
Where has gitk been all my life!? Very useful tool, thanks for suggesting

This should be removed now